emcc.py

```diff
@@ -2338,11 +2338,11 @@ def get_clang_command_asm(src_file):
     headers = [header for _, header in input_files]
     for header in headers:
       if not header.endswith(HEADER_ENDINGS):
-        exit_with_error('cannot mix precompile headers with non-header inputs: ' + str(headers) + ' : ' + header)
+        exit_with_error(f'cannot mix precompile headers with non-header inputs: {headers} : {header}')
```

Comment:
```suggestion
        exit_with_error(f'cannot mix precompiled headers with non-header inputs: {headers} : {header}')
```
Not sure if we want to fix typos here, too.